recipes/ironcub-models/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+cmake -B build -S . \
+	   -GNinja  \
+     -DCMAKE_INSTALL_PREFIX=$PREFIX \
+	   -DCMAKE_BUILD_TYPE=Release \
+       ${CMAKE_ARGS}
```

Comment:
```suggestion
#!/bin/sh

cmake -B build -S . \
	   -GNinja  \
	   -DCMAKE_BUILD_TYPE=Release \
       ${CMAKE_ARGS}
```
This shouldn't be needed when `${CMAKE_ARGS}` is specified.

Reply:
@danielnachun see the comment in https://github.com/conda-forge/staged-recipes/pull/30690#discussion_r2335743711 .